Server/Content/Sources/App/Controllers/TextController.swift

```diff
@@ -0,0 +1,29 @@
+import Fluent
+import Vapor
+
+struct TextController {
+    func all(req: Request) throws -> EventLoopFuture<[Text]> {
```

Comment:
Let's use full names for variables `req` -> `request`
The same is for the rest of the methods in this class

Reply:
```swift
 func all(req: Request) throws -> EventLoopFuture<[Text]> {
```
and
```swift
func getById(req: Request) throws -> EventLoopFuture<Text> {
```
Basically have the same behavior of returning something to the user, but why one of them has prefix `get` while other don't? We need to have consistent naming for the similar behavior